rs/bitcoin/ckbtc/minter/src/lifecycle/init.rs

```diff
@@ -7,7 +7,7 @@ use ic_btc_interface::Network;
 use serde::Serialize;
 
 pub const DEFAULT_MIN_CONFIRMATIONS: u32 = 6;
-pub const DEFAULT_KYT_FEE: u64 = 1000;
+pub const DEFAULT_CHECK_FEE: u64 = 1000;
```

Comment:
What? Why was the default KYT fee 1000?   :thinking: 
Anyway, we should set the default check fee to 100, right?